lib/octobox/configurator.rb

```diff
@@ -163,5 +145,15 @@ def github_admin_ids
       return @admin_github_ids = [] unless admin_github_ids.present?
       @github_admin_ids = admin_github_ids.split(',')
     end
+
+    private
+
+    def env_boolean(env_var_name)
+      ENV[env_var_name].try(:downcase) == 'true'
```

Comment:
it would be nice if this was:
```ruby
%w(true 1).include?(ENV[env_var_name].try(:downcase))
```
Because we had `ENV['env'].present?`, it's possible people used other values. 1 or true seem reasonable to me?